app/cosi/README.md

```diff
@@ -0,0 +1,338 @@
+[![Build Status](https://travis-ci.org/dedis/cothority.svg?branch=master)](https://travis-ci.org/dedis/cothority)
```

Comment:
I started to update the wiki on a non-regular basis - should this be kept as a README or kept _only_ in the wiki at https://github.com/dedis/cothority/wiki/CoSi ?